discovery-infra/test_infra/helper_classes/kube_helpers/agent_cluster_install.py

```diff
@@ -1,24 +1,17 @@
 import logging
-
-import waiting
-
-from typing import Union, Optional, Tuple
-from pprint import pformat
-
 from base64 import b64decode
+from pprint import pformat
+from typing import Optional, Tuple, Union
 
+import waiting
 from kubernetes.client import ApiClient, CustomObjectsApi
 
-from tests.conftest import env_variables
+from ... import consts
```

Comment:
... ? bad path

Reply:
At the moment, consts package is under `test_infra`  so it's imported this way.
When changing env_variables, rearrange and separate consts we might change the package path